### PR TITLE
refactor(lambda): use lowercase boolean for type declarations

### PR DESCRIPTION
**Pull request by yodeee9**
Updated the type annotations to use the primitive `boolean` type instead of the object wrapper `Boolean`. 
This change aligns with TypeScript's recommended naming conventions and provides clearer type indications.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by mrgrain — APPROVED** [review]
@yodeee9 Please don't manually update the branch, that breaks the automation. Once a PR is approved it's best to just leave it unless it obviously stalled and maintainers don't notice it.

**Comment by yodeee9** [comment]
Thank you for merging my pull request! I'll make sure not to manually update in the future.

**Pull request merged**
